src/SFA.DAS.ApplyService.InternalApi/Controllers/GatewayOrganisationChecksController.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SFA.DAS.ApplyService.Application.Apply.Roatp;
+using SFA.DAS.ApplyService.InternalApi.Infrastructure;
+
+namespace SFA.DAS.ApplyService.InternalApi.Controllers
+{
+    [Authorize]
+    public class GatewayOrganisationChecksController : Controller
```

Comment:
Are there tests for this code?

Reply:
Seems not, I thought there were.  I'll put some in.